Compiler warning regarding literal operator suffixes not preceded by '_' being reserved

Post 864bcd125e6b5374d9170d687c5be59c243072dd, I'm seeing the following compiler warning:

```
external/drake/systems/framework/system_base.cc: In member function 'virtual std::__cxx11::string drake::systems::SystemBase::GetSystemPathname() const':
external/drake/systems/framework/system_base.cc:20:39: warning: literal operator suffixes not preceded by '_' are reserved for future standardization
   using std::string_literals::operator""s;
```
Can this warning be addressed or suppressed?

--- a/systems/framework/system_base.cc
+++ b/systems/framework/system_base.cc
@@ -17,10 +17,12 @@ namespace systems {
 SystemBase::~SystemBase() {}
 
 std::string SystemBase::GetSystemPathname() const {
-  using std::string_literals::operator""s;
+  // NOLINTNEXTLINE(build/namespaces): using operator""s issues a warning.
+  using namespace std::string_literals;
+
   const std::string parent_path =
-    get_parent_service() ? get_parent_service()->GetParentPathname()
-                         : ""s;
+      get_parent_service() ? get_parent_service()->GetParentPathname()
+                           : ""s;
   return parent_path + "::"s + GetSystemName();
 }
 

